play/plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/internal/PlayApiService.kt

```diff
@@ -69,6 +79,18 @@ internal abstract class PlayApiService @Inject constructor(
         fileOps.delete { delete(editIdFileAndFriends) }
     }
 
+    override fun close() {
+        if (cleanupAtClose) {
+            cleanup()
+        }
+    }
+
+    override fun onFinish(event: FinishEvent) {
+        if (event is TaskFinishEvent) {
+            tasksHasFailed = tasksHasFailed || (event.result is TaskFailureResult)
```

Comment:
|=

Reply:
Yeah I know, but it does not exist in Kotlin:
https://kotlinlang.org/docs/keyword-reference.html#operators-and-special-symbols